kernel/src/collections/list_v.rs

```diff
@@ -0,0 +1,606 @@
+#![feature(const_refs_to_cell)]
+
+use vstd::prelude::*;
+
+verus! {
+
+use vstd::cell::*;
+
+pub struct ListLinkV<'a, T: 'a + ?Sized>(pub PCell<Option<&'a T>>);
+
+impl<'a, T: ?Sized> ListLinkV<'a, T> {
+    pub const fn empty() -> (res: (ListLinkV<'a, T>, Tracked<PointsTo<Option<&'a T>>>))
+        ensures
+            res.0.0.id() == res.1@.id(),
```

Comment:
It is also because we are returning a tuple. I cannot write the return type as `(res_list: ListLinkV<'a, T>, res_points_to: Tracked<PointsTo<Option<&'a T>>>)` as it will not compile. So I am forced to use `res.0` to refer to the list and `res.1` for the points_to struct.